--- dante-1.4.1/lib/upnp.c.orig	2014-08-21 18:21:50.000000000 +0200
+++ dante-1.4.1/lib/upnp.c	2014-10-25 18:12:07.582908742 +0200
@@ -828,7 +828,7 @@
 {
    const char *function = "sighandler()";
 
-   slog(LOG_DEBUG, function);
+   slog(LOG_DEBUG, "%s", function);
    upnpcleanup(-1);
 
    /* reinstall original signal handler. */
@@ -936,7 +936,7 @@
 {
    const char *function = "atexit_upnpcleanup()";
 
-   slog(LOG_DEBUG, function);
+   slog(LOG_DEBUG, "%s", function);
    upnpcleanup(-1);
 }
 #endif /* SOCKS_CLIENT */
