diff -urNp -x '*.orig' dante-1.4.2.org/configure.ac dante-1.4.2/configure.ac
--- dante-1.4.2.org/configure.ac	2016-06-25 01:01:33.000000000 +0200
+++ dante-1.4.2/configure.ac	2023-02-02 08:59:23.291445806 +0100
@@ -71,7 +71,7 @@ esac
 
 AM_INIT_AUTOMAKE(dante, ${version})
 AC_CONFIG_SRCDIR(include/common.h)
-AM_CONFIG_HEADER(include/autoconf.h)
+AC_CONFIG_HEADERS(include/autoconf.h)
 
 AC_CONFIG_MACRO_DIR([m4])
 
